.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
That's a weird shebang ❓

Isn't it supposed to be `/bin/bash` ?

Also, you don't really need it as you run the script by specifying what binary to execute it. 
`bash ./script`